faq.html

```diff
@@ -38,10 +38,11 @@
       </div>
       <div class="nav-login">
         <a
+          tabindex="1"
           class="btn-login"
           href="https://github.com/login/oauth/authorize?client_id=23c78f66ab7964e5ef97"
         >
-          <button class="btn-login-text">
+          <button tabindex="1" class="btn-login-text">
```

Comment:
button itself is a semantic element and we should not change the order of tabIndex. Can you provide the reason for changing tabindex order?

Reply:
removed tabindex from button